tldw_Server_API/app/api/v1/endpoints/mcp_unified_endpoint.py

```
@@ -1595,6 +1595,52 @@ async def health_check(
+    """Test connectivity to an external MCP server URL."""
+    import httpx
+
+    headers: dict[str, str] = {}
+    if req.auth_type == "bearer" and req.secret:
+        headers["Authorization"] = f"Bearer {req.secret}"
+    try:
+        async with httpx.AsyncClient(timeout=10.0) as http:
+            resp = await http.get(req.url, headers=headers)
+            resp.raise_for_status()
+            return MCPConnectionTestResponse(reachable=True)
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)